roxctl/declarativeconfig/create/access_scope.go

```diff
@@ -0,0 +1,216 @@
+key, value, operator. The key value pairs are expected to be separated with ;.
+Each tuple represents a requirement, which will be used to construct the label selector.
+You may specify this flag multiple times, to create a conjunction of requirements which should apply for a label selector
+to match.
+
+Example of a label selector requiring values: --cluster-label-selector "key=kubernetes.io/hostname;operator=IN;values=nodeA,nodeB"
+Example of a label selector not requiring values: --cluster-label-selector "key=custom-label;operator=EXISTS"
+
+NOTE: The created access scope will only contain a single label selector, where each specified requirement
+will be in conjunction. If you desire to create multiple label selectors, you have to adjust the YAML output manually.
+`
+
+func accessScopeCommand(cliEnvironment environment.Environment) *cobra.Command {
+	accessScopeCmd := accessScopeCmd{accessScope: &declarativeconfig.AccessScope{}, env: cliEnvironment}
+
+	cmd := &cobra.Command{
+		Use:  accessScopeCmd.accessScope.Type(),
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := accessScopeCmd.Validate(); err != nil {
+				return err
+			}
+			return accessScopeCmd.PrintYAML()
+		},
+	}
+
+	cmd.Flags().StringVar(&accessScopeCmd.accessScope.Name, "name", "", "name of the access scope")
+	cmd.Flags().StringVar(&accessScopeCmd.accessScope.Description, "description", "",
+		"description of the access scope")
+
+	cmd.Flags().Var(&includedObjectsFlag{includedObjects: &accessScopeCmd.accessScope.Rules.IncludedObjects}, "included",
+		`list of clusters and their namespaces that should be included within the access scope.
+In case all namespaces of a specific cluster should be included, specify --included cluster-name.
+In case only a subset of namespace should be included, specify --included cluster-name=namespaceA,namespaceB`)
+
+	// Currently, its only support to provide a single cluster-label-selector for the access scope.
+	// The reason is of the complexity of the resulting struct, its currently not possible to associated N requirements
+	// with M label selectors (cluster or namespace). Hence, the command line option only currently allows to create
+	// a single one, with the hint that advanced users may adjust the output YAML to include additional label selectors
+	// if they wish to do so.
+
+	cmd.Flags().Var(&requirementFlag{requirements: &accessScopeCmd.clusterRequirements}, "cluster-label-selector",
+		labelSelectorUsage)
+
+	cmd.Flags().Var(&requirementFlag{requirements: &accessScopeCmd.namespaceRequirements}, "namespace-label-selector",
+		labelSelectorUsage)
+
+	utils.Must(cmd.MarkFlagRequired("name"))
+
+	return cmd
+}
+
+type accessScopeCmd struct {
+	accessScope *declarativeconfig.AccessScope
+	env         environment.Environment
+
+	clusterRequirements   []declarativeconfig.Requirement
+	namespaceRequirements []declarativeconfig.Requirement
+}
+
+func (a *accessScopeCmd) Validate() error {
```

Comment:
Should we validate if used names are valid? Eg. if labels used in label selectors match k8s requirements?